packages/test-react-app/src/veramo/setup.ts

```diff
@@ -107,7 +107,7 @@ export function getAgent(options?: IAgentOptions): TAgent<InstalledPlugins> {
                 rpcUrl: 'https://goerli.infura.io/v3/' + INFURA_PROJECT_ID,
               },
               {
-                chainId: 421613,
+                chainId: BigInt(421613),
```

Comment:
Please revert the use of BigInt in configs
```suggestion
                chainId: 421613,
```